install/gcp/installer/forseti_installer.py

```diff
@@ -199,8 +199,9 @@ def preflight_checks(self):
                                          is_cloudshell)
         self.organization_id = gcloud.lookup_organization(self.project_id)
         self.config.generate_identifier(self.organization_id)
-        self.check_if_authed_user_in_domain(
-            self.organization_id, authed_user)
+        if not self.config.skip_check_authed_user:
```

Comment:
If we are using a service account to auth gcloud, shouldn't the authed user become the service account in this case? Is there any reason we need to skip this?

Reply:
That makes sense, so instead of the param --skip-check-authed-user, the check depends on if there's a service account activated? I mean, delete this param and only check if user belongs to domain when there's no a service account identified? @joecheuk